templates/4a_compensate_image_data.ipynb

```diff
@@ -5,9 +5,26 @@
    "id": "ca8b60ef",
```

Comment:
<p>Change this to be compatbile with new automated panel adjustment script and empty json file detection script</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
the empty json PR #187 is merged in. I will handle adding the panel conversion changes to this notebook in PR #202.

Reply:
@camisowers So this looks good to merge in? I'll add the link to the directory structure section in the read me for #190.

Reply:
<p>I'm working on a function automatically copy over extracted images instead of re-extracting from bin files, so the empty json check won't matter. I can add the readme link in PR #190 no worries</p>

Reply:
Great, sounds swell.